icu4c/source/i18n/measunit_extra.cpp

```diff
@@ -631,7 +650,27 @@ compareSingleUnits(const void* /*context*/, const void* left, const void* right)
 
 } // namespace
 
+int32_t umeas_getPrefixPower(UMeasurePrefix unitPrefix) {
```

Comment:
Please include the C API boilerplate.

```suggestion
U_CAPI int32_t U_EXPORT2
umeas_getPrefixPower(UMeasurePrefix unitPrefix) {
```

Reply:
Done though I do suspect the C APIs won't be very useful, given that there's no C API that's actually making use of this enum? Leaving unresolved...